other/iris/iris.jl

```diff
@@ -18,7 +18,7 @@
 # To run this example, we need the following packages:
 
 using Flux
-using Flux: logitcrossentropy, normalise, onecold, onehotbatch
+using Flux: logitcrossentropy, normalise, onecold, onehotbatch, params
```

Comment:
I would prefer either both or neither of `logitcrossentropy` and `params` be used qualified or imported. Currently, this imports both but only uses one.

Reply:
This I would keep, the below is then not necessary.

Is it what you suggest. Also depends on the comment above, what is the way how to edit/update. :)